packages/admin/admin-generator/src/commands/generate/generate-command.ts

```diff
@@ -184,7 +185,26 @@ export type GridConfig<T extends { __typename?: string }> = {
     selectionProps?: "multiSelect" | "singleSelect";
 };
 
-export type GeneratorConfig<T extends { __typename?: string }> = FormConfig<T> | GridConfig<T> | TabsConfig<T>;
+type CrudPageGridConfig = {
+    component: ComponentType;
+};
+
+export type CrudPageFormConfig = {
+    component: ComponentType;
+    pageTitle?: string;
+};
+
+export type CrudPageConfig<T extends { __typename?: string }> = {
+    type: "crudPage";
+    topLevelTitle?: string;
+    gqlType: T["__typename"];
+    grid: CrudPageGridConfig;
+    forms?: CrudPageFormConfig;
```

Comment:
why forms, when you specify only a single component?

Reply:
It's used to configure both forms at once, `add` and `edit` so I thought plural made sense.   

Would you still prefer `form`? 
_I don't have a strong opinion on that 😄_